sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentViewModel.kt

```diff
@@ -117,7 +116,7 @@ internal class AuthorizingPaymentViewModel(
     }
 
     private suspend fun sendAuthenticationRequest(netceteraConfig: NetceteraConfig) {
-        val transaction = threeDS2Service.createTransaction(netceteraConfig.directoryServerId!!, netceteraConfig.messageVersion)
+        val transaction = threeDS2Service.createTransaction(netceteraConfig.directoryServerId!!, netceteraConfig.messageVersion!!)
```

Comment:
Does "!!" cause crash if value is NULL? If so it's always recommended to avoid it as some future changes could break it

Reply:
It will throw an error that the param is null if it is missing from the response then it will open webview which is the expected behavior.

Reply:
Emvco recommends to use the latest 3ds protocol version if Message version is null

Reply:
![image (69)](https://github.com/omise/omise-android/assets/122003737/a4d48253-f1d9-45bc-bec2-1c8325e66a9d)

Reply:
but what if our MS does not support the latest version, then it will fail. It is not safe to leave it to Netcetera defaults.

Reply:
@annatotest Any other concerns ?